docs/features/payment-management/pay-your-contacts.md

```diff
@@ -23,12 +23,19 @@ Since April 2022 you can also download a CSV file for all contacts that have an
 
 ## Bulk payments
 
-Hotspotty gives you the option to pay multiple people at once so that you do not have to worry about copy/pasting every single HNT wallet address. We have 2 options to quickly have your contacts paid:
+Hotspotty gives you the option to pay multiple people at once so that you do not have to worry about copy/pasting every single HNT wallet address. We have 4 options to quickly have your contacts paid:
 
 ### Pay using a QR code
 
 You can make bulk payments by simply scanning a QR code provided when you [create a commission report](../payment-management/generate-commission-reports). Open up the Helium app. Go to 'My Wallet', and click on the blue 'Send HNT' button. Instead of entering the recipient address you can easily click on the qr code symbol, and you are good to go.
 
+### Pay using Hotspotty on your phone
+
+To use this option, you are required to have the new Helium wallet app installed. The limit of paying 25 hosts at once using the QR code can be exceeded using this option up to an unlimited amount of transactions. Simply open up the desired commission report on your phone. There will be an additional option that's not visible on the laptop/desktop version. <button className="hotspotty-button">Pay with Helium wallet</button> This will open up the new Helium wallet app and ask you to sign the transactions. Additionally, you can pay individual contacts with this method as well by manually registering payments for them. [Click here to learn more](../planning-and-optimizing/manage-contacts#register-payments).
```

Comment:
Can you please rename "Helium wallet app" to "Helium HNT Wallet app (iOS or Android)" with the following links:
https://apps.apple.com/us/app/helium-hnt-wallet/id1609525848
https://play.google.com/store/apps/details?id=com.helium.wallet.app

Reply:
"The limit of paying 25 hosts at once using the QR code can be exceeded using this option up to an unlimited amount of transactions." -> "There is no limit to the number of contacts that can be paid per transaction, unlike the QR code payment option (max 25 per QR)."